### PR TITLE
Add a blogpost about Active Record #pluck method

### PR DESCRIPTION
**Pull request by shadre**

### CODE REVIEW

File: _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Knowing we only really need to load the data from the `email` column, we could improve the above by chaining the Active Record's `#select` method:
+
+{% highlight ruby %}
+User.all.select(:email).map(&:email)
+# SELECT "users"."email" FROM "users"
+
+#=> ["kate@example.com", "john@example.org", "janet@example.com"]
+{% endhighlight %}
+
+The SQL query is limited to one column now, but the `ActiveRecrod` objects are still being created in the intermediate step before `map`. Granted, they contain less data now (as the attributes we didn't `select` are omitted), but it's still quite unnecessary in this use case.
```
Comment (ArturT) [inline _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md:39]: step before _the_ map

File: _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+author: "Shadi Rezek"
+categories: techtips ruby rails
+og_image: ""
+---
+
+Have you heard of the [#pluck](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck){:target="_blank"} method from Rails' Active Record? Read on to see what it does and how it can be used to easily query single columns from your database!
+
+## The use case
+
+Sometimes when retrieving data from the database, we are only interested in certain attributes (e.g. just emails of all the users, instead of data residing in all columns of the `users` table).
```
Comment (ArturT) [inline _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md:14]: missing comma after `e.g.,`

File: _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+The _naïve_ approach would be to load all the data into the memory, and then `map` it so that only the subset of the data is left, e.g.:
```
Comment (ArturT) [inline _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md:16]: comma after memory might be unnecessary here

File: _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md (hunk 1 above)
Comment (ArturT) [inline _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md:39]: typo in ActiveRecord

**Review by ArturT — COMMENTED** [review]
The article looks good. Simple to explain the pluck method concept :)

File: _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md (hunk 1 above)
Comment (shadre) [reply _posts/2021-04-17-querying-single-columns-in-rails-active-record-using-pluck-method.md:39]: Thanks. I got the same suggestion from grammarly, but frankly I don't agree with it in this case. ;-)

**Pull request merged**